tsp-core/src/main/java/pl/edu/agh/iet/tsp/core/config/DatabaseConfig.java

```diff
@@ -0,0 +1,51 @@
+package pl.edu.agh.iet.tsp.core.config;
```

Comment:
cyliczne zaleznosci. domene mamy bardzo prosta i kodu zwiazanego z nia, daosami i serwisami bedzie bardzo niewiele, nie wiem czy jest sens to rozdzielac.
ale jesli bysmy to rozdzielili to trzeba by zrobic 3 moduly: domain, db i service; db zalezy od domain, service zalezy od obydwu pozostalych. bez modulu service mialbys cykliczna zaleznosc: core zalezy od db (bo ma serwisy) i db zalezy od core (w oczywisty sposob).
no i jeszcze trzeba by dodac jakis modul "web" w ktorym beda kontrollery. "TrueSocialPlatform.java" czyli nasz main nie powinien byc w core.
ogarne to tak ze bedzie dobrze, tylko daj znac czy chcemy to rozrabniac na 4 moduly (domain, db, service, web), czy tylko na core + web